src/nls/root/strings.js

```diff
@@ -601,7 +602,8 @@ define(function (require, exports, module) {
                 RECENT_DOC: "Recent Documents",
                 DOCUMENT: "Documents",
                 MENU_COMMAND: "Menu Commands",
-                LAYER: "Layers",
+                ALL_LAYER: "Layers",
+                CURRENT_LAYER: "Layers",
```

Comment:
I imagine they wouldn't both be used in the same search module, so I think we can get away with just saying that you are searching for layers.